monolog/monolog/2014-12-29-1.yaml

```diff
@@ -0,0 +1,8 @@
+title:     Fixed header injection in NativeMailerHandler
```

Comment:
The title should not contain ``Fixed``. The vulnerability is not the fix itself.